website/docs/Basic_operations/create.md

```diff
@@ -0,0 +1,44 @@
+---
+sidebar_position: 2
+---
+
+# Insert operation
+
+The insert operation adds a new document to a collection.
+
+## Insert a single document
+
+The `insertOne()` command is used to add a single document into a collection, using this syntax format:
+
+```sh
+db.collection.insertOne({field1: value1, field2: value2,.... fieldN: valueN})
+```
+
+The example below depicts how a single document can be added to a collection.
+
+```sh
+db.scientists.insertOne({name:{firstname: "Thomas", lastname: "Edison"}, born: 1847, invention: "lightbulb"})
+```
+
+If the operation is successful, you will get a response with acknowledged set to true, and the autogenerated ObjectId of the document.
+
+:::note
+If a collection does not exist, the insert command automatically creates one.
+:::
```

Comment:
This one doesn't format as markdown, some ideas how to format it here: https://stackoverflow.com/questions/25654845/how-can-i-create-a-text-box-for-a-note-in-markdown

Reply:
Thanks, Patryk. This works for Markdown in Docusaurus since they have packages for handling that, but I'll look into this example too.

Reply:
Oh okay then no problem with that :)

Reply:
https://docusaurus.io/docs/markdown-features/admonitions